conda/core/index.py

```diff
@@ -171,6 +171,19 @@ def _supplement_index_with_system(index):
         rec = _make_virtual_package('__' + libc_family, libc_version)
         index[rec] = rec
 
+    for flag in context.cpu_flags:
```

Comment:
Do we need a whitelist or name mapping potentially?
I don't know whether there are any conflicting flag names between different CPU vendors/architectures, e.g., whether feature `xyz` exists for Intel/AMD x86 but also for ARM/PPC but with a different meaning.

Reply:
If they have a different meaning, so be it. I don't see any reason to not allow conflicting feature names if they are in different subdirs.